scripts/docker-compose/install.sh

```diff
@@ -115,7 +115,13 @@ case $yn in
 		exit 1;;
 esac
 
-sudo -E docker-compose --parallel 1 pull
+services=$(sudo -E docker-compose config --services)
```

Comment:
What's the reason for this change?

Reply:
The `--parallel` argument seems not to exist anymore:

![Screenshot 2024-11-26 at 18 09](https://github.com/user-attachments/assets/027c819b-8c80-4bcf-91b3-31e8ca69f5f0)

Reply:
You can find more infos in this Slack thread: https://openreplay.slack.com/archives/C020MT0LK18/p1732632392272919